egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -357,6 +357,37 @@ def parse_generic_config_vars_file(var_file):
     raise Exception('Error while parsing the file {0}'.format(var_file))
 
 
+def parse_input_model(input_model):
```

Comment:
rename to get_input_model_info, because it doesn't parse the model (it's not  a text file).
The documentation is very bad: it doesn't even explain what the function returns.  It says
it "outputs" the left and right context.  I don't see it opening any files to output it.  Think carefully
about the meaning of the words you use.